src/olympia/addons/api_urls.py

```diff
@@ -0,0 +1,12 @@
+from django.conf.urls import patterns, url
+
+from . import views
```

Comment:
I'm okay with not moving them and maybe finalizing the discussion in the other ticket first